src/chunk.c

```diff
@@ -4855,3 +4861,12 @@ ts_chunk_get_osm_chunk_id(int hypertable_id)
 
 	return chunk_id;
 }
+
+/* when a tiered chunk is created, it is assigned a range [max, infinity)
+ XXX could this potentially be a valid range? */
+bool
+ts_osm_chunk_range_is_invalid(Oid dimtype, int64 range_start, int64 range_end)
+{
+	int64 dim_max = ts_time_get_max(dimtype);
+	return ((range_start == dim_max) && (range_end > range_start));
+}
```

Comment:
I dont think this reflects the chunks created by osm because they take chunk_time_interval into consideration. Maybe we should check for range_end == INT64_MAX